app/customer_routes.py

```
@@ -0,0 +1,123 @@
+from flask import Blueprint, request, make_response, jsonify
+from app import db 
+from app.models.customer import Customer
+from app.models.video import Video 
+from app.models.rental import Rental
+from dotenv import load_dotenv
+from datetime import datetime
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+load_dotenv()
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def get_customers():
+
+    customers = Customer.query.all()
+    customers_respone = []
+
+    if customers is None:
+        return 404
```

If there are no customers, query.all() will return an empty list, unlike query.get() which does return None.  This check will always fail, if you want to return an error message in the case of no customers, the conditional needs to check for an empty list or falsey.